lib/hybrid.h

```diff
@@ -452,6 +453,7 @@ struct HybridSimulator final {
       const auto& gatex = gatexs[i];
 
       if (gatex.schmidt_bits == 0) {
+        // Continue if gatex has Schmidt rank 1.
```

Comment:
For clarity, maybe replace `Continue if...` with `Skip decomposition of gatex if...`